db/migrate/20240511212107_add_price_to_store_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriceToStoreItems < ActiveRecord::Migration[7.1]
+  def change
+    add_column :store_items, :price, :integer
```

Comment:
Shouldn't `price` be a `float` instead of an `integer`?

Reply:
I've made the change